03.bowling/bowling.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'rubygems'
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0 if shots.count <= 17
+  else
+    shots << s.to_i
+  end
+end
+
+last_frame = shots.slice!(18..)
+frames = shots.each_slice(2).to_a
+frames.push(last_frame)
+
+point = 0
+frames.each_with_index do |frame, i|
```

Comment:
```suggestion
point = frames.each_with_index.sum do |frame, i|
```

sumメソッドを使うとブロックの内部が少しシンプルになります